### ISSUE TITLE
Change Item List Easily

### ISSUE THREAD

**Issue by conor-f** [go-vikunja/frontend#40]
It's quite a few clicks to move an item from one list to another. My ideal workflow is to put everything to a default list, then be able to click and drag that item into a new list, most likely in the sidebar.

Does this make sense? Can post screenshots if it would help explain

**Comment by kolaente** [comment]
Yeah the idea makes sense.

It will be a bit of work so I don't see it happening soon, but I've just added it to the backlog.

**Comment by conor-f** [comment]
Appreciated! Thanks for that :)

**Comment by dpschen** [comment]
For me moving to a temporary list seems like an unnecessary and complex in-between step.

@conor-f:
Would it solve your use case if you would be able to drag tasks in other lists in the sidebar directly.

**Comment by conor-f** [comment]
Yeah, really it's just a means of reorganization. I frequently end up putting things my default list instead of using the appropriate `#` value to put it in the correct list (e.g. "Today's Todo List") and then at the end of my day I want to move things from "Today's Todo List" to the relevant longer term one (e.g. Move groceries I didn't pick up that were in "Today's Todo List" into the "Shopping" todo list).

My use case would most often be opening up one list, looking at the items in it, and dragging them from the list to a different list I can see in the Sidebar in order to move them to that list.

Does that make sense?

**Comment by dpschen** [comment]
Makes sense!

@kolaente can you link & adjust the backlog item?

**Comment by ohare93** [comment]
To expand on this request (since I want something similar) it seems to me there are multiple levels of achieving this feature request. Here I have listed a few, with the higher the number being both easier for the user and more effort to implement (imo, as a programmer who has not looked at the source code for this repo). The below options presume you start inside the list view.

0. Current functionality: Open a task, select move (or hit m), type in the name of the list (with prediction), hit enter. Repeat these 4 steps for every other task you wish to move... :confused:
1. Open a task with subtasks / related tasks, hit a new Move (with relations) button, select which type of relations to move with the original task, type in the name of the list (with prediction), hit enter
2. Select multiple tasks, hit a (new) move button, type in the name of the list (with prediction), hit enter. (This would be *good enough* for me :+1:)
3. Drag and drop a single task into the sidebar of lists (what @conor-f suggested / requested). Repeat this 1 step for each task you wish to move :+1: 
4. Select multiple task in a list, then drag and drop them into the sidebar of lists :+1: :+1: 


Then to add in a wildcard that I have no idea where it would be in the complexity list, what I would reaaaaaally like: **Bucket mode!** (Mass Move Selector) :heart_eyes:  To be able to go into this "Bucket Mode" (via a button or a shortcut) which first asks you which list you wish to move things to, then you simply click on an item to move it instantly. Select 1 or multiple. Change the destination list at any time. (Feature creep: set keyboard shortcuts to change the list while in Bucket mode for the number keys (1 is list Incoming, 2 is list Today, 3 is list AfterKidsAreAsleep, etc). Exit "Bucket mode" at any time.

**Comment by Torqu3Wr3nch** [comment]
+1.

This would be a major UX improvement to Vikunja. Much more efficient for those who use a "GTD"-like workflow where tasks are dumped into the inbox and then sorted/triaged to other lists/projects later.
